src/components/Modal/Modal.vue

```diff
@@ -769,7 +776,7 @@ $header-size: 50px;
 			max-width: initial;
 			width: 100%;
 			max-height: initial;
-			height: calc(100% - $header-size);
+			height: calc(100% - 50px);
```

Comment:
For example, we dropping `$header-size` ?

Reply:
see https://github.com/nextcloud/nextcloud-vue/pull/2197#discussion_r699448026